specifications/catalog/example/dcat.distribution.example.option1.json

```diff
@@ -0,0 +1,123 @@
+{
+    "@context":  "https://w3id.org/idsa/v5/context.json",
+    "@id": "http://provider.com/catalog1/dataset1/",
+    "@type": "dcat:Dataset",
+    "dct:title" : "IDS Dataset Collection",
+    "dct:description": [ {
+        "@language": "en",
+        "@value": "This dataset includes all assets which identifiers comply to the pattern 'urn:guid:[A-F0-9]{8}\\-[A-F0-9]{4}\\-[A-F0-9]{4}\\-[A-F0-9]{4}\\-[A-F0-9]{12}'. A download via a S3 dataplane costs 5 EUR, while the HTTP_REST dataplane option costs only 2 EUR. The consumer must be a TRUST_PLUS certfified connector. No further distribution is allowed."
+    }, {
+        "@language": "de",
+        "@value": "Dieses Dataset bezeichnet alle Assets, deren Identifier dem Pattern 'urn:guid:[A-F0-9]{8}\\-[A-F0-9]{4}\\-[A-F0-9]{4}\\-[A-F0-9]{4}\\-[A-F0-9]{12}' folgen. Ein Abruf über eine S3 Dataplane kostet 5 EUR und über eine HTTP_REST Dataplane 2 EUR. Der Consumer muss ein TRUST_PLUS zertifizierter Connetor sein. Eine Weiterverbreitung ist nicht gestattet."
+    } ],
+    "dcat:keyword": [
+        "IDS",
+        "data asset",
+        "dcat dataset",
+        "collection"
+    ],
+    "dcat:distribution": [ {
+        "@type": "dcat:Distribution",
+        "dct:title" : "Distribution#1",
+        "dct:description": "This is a distribution everyone can see.",
+        "dcat:keyword": [
+            "dataplane",
+            "public",
+            "distribution"
+        ],
+        "ids:dataPlaneKind": {
+            "@id": "idsc:HTTP_REST"
+        },
+        "dcat:accessService": [ {
+            "@type": [ "dcat:DataService", "idsc:NEGOTIATION_SERVICE" ],
+            "dcat:endpointURL": {
+                "@type": "xsd:anyURI",
+                "@value": "https://provider.com/connector/negotiation"
+            }
+        }, {
+            "@type": [ "dcat:DataService", "idsc:TRANSFER_SERVICE" ],
+            "dcat:endpointURL": {
+                "@type": "xsd:anyURI",
+                "@value": "https://provider.com/connector/transfer"
+            }
+        } ]
+    }, {
+        "@type": "dcat:Distribution",
+        "dct:title" : "Distribution#2",
+        "dct:description": "This is another distribution.",
+        "dcat:keyword": [
+            "dataplane",
+            "public",
+            "distribution"
+        ],
+        "ids:dataPlaneKind": {
+            "@id": "idsc:S3"
+        },
+        "dcat:accessService": [ {
+            "@type": [ "dcat:DataService", "idsc:NEGOTIATION_SERVICE" ],
+            "dcat:endpointURL": {
+                "@type": "xsd:anyURI",
+                "@value": "https://provider.com/connector/negotiation"
+            }
+        }, {
+            "@type": [ "dcat:DataService", "idsc:TRANSFER_SERVICE" ],
+            "dcat:endpointURL": {
+                "@type": "xsd:anyURI",
+                "@value": "https://provider.com/connector/transfer"
+            }
+        } ]
+    } ],
+    "dcat:hasPolicy": {
```

Comment:
I would approach this differently because it muddies what a distribution is, and it will be difficult to implement software that can reasonably process this. A distribution defines:


1. An endpoint to initiate a contract negotiation for the asset
2. An endpoint to initiate a transfer request for the asset
3. One or more supported transport protocols for the asset


Given this definition, distribution and policy are orthogonal; only an asset (data set) has a policy. For example, the same transport protocol can be used if an asset is restricted to Europe or it is not. Different "data planes" may be needed, but per my previous comment, a data plane is an implementation detail and not reducible to a transport protocol. 

Let's take the example of S3 and Europe. In this case, the data plane is the same (it is the AWS infrastructure that implements the S3 protocol) if an asset is restricted to Europe or it can be hosted in the U.S. The only thing that changes is the way the data plane is used. An S3 EU region must be selected for an asset restricted to Europe when transferring data.

This would still hold when a data plane is not cross-region. For example, assume a provider has one data plane supporting HTTP/REST in Europe and one in the US. The selection of the data plane is an implementation detail since the transport protocol (HTTP/REST) remains the same. The selection can be based on the policy associated with the asset.

Another way of looking at this is that the consumer chooses which "policy" it wants by selecting an asset and one of its contained offers. It selects which transport protocol it wants to use by selecting one of the asset's distributions. A consumer does not choose the provider's data plane. 

Going back to the case where a provider wants to make the same data available to consumers at different price points based on download speed, this should be modeled as two different assets. In many cases, the transport protocol would remain the same, e.g., HTTP/REST, with the difference being reserved bandwidth capacity. This approach would also be much easier for implementations to process than a policy that uses XONE constraints and requires implicit knowledge linking the constraint type to the distribution.